spec.html

```diff
@@ -28132,278 +28130,159 @@ <h1>Function Properties of the Math Object</h1>
-          <li>
-            If _y_ is *+&infin;* and _x_ is *+&infin;*, the result is an implementation-approximated value representing +&pi; / 4.
-          </li>
-          <li>
-            If _y_ is *+&infin;* and _x_ is *-&infin;*, the result is an implementation-approximated value representing +3&pi; / 4.
-          </li>
-          <li>
-            If _y_ is *-&infin;* and _x_ is *+&infin;*, the result is an implementation-approximated value representing -&pi; / 4.
-          </li>
-          <li>
-            If _y_ is *-&infin;* and _x_ is *-&infin;*, the result is an implementation-approximated value representing -3&pi; / 4.
-          </li>
-        </ul>
+        <p>When the `Math.atan2` method is called with arguments _x_ and _y_, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _ny_ be ? ToNumber(_y_).
+          1. Let _nx_ be ? ToNumber(_x_).
+          1. If _ny_ is *NaN* or _nx_ is *NaN*, return *NaN*.
+          1. If _ny_ &gt; 0, then
+            1. If _nx_ is *+0* or _nx_ is *-0*, return an implementation-approximated value representing +&pi; / 2.
+          1. If _ny_ is *+0*, then
+            1. If _nx_ &gt; 0 or _nx_ is *+0*, return *-0*.
+            1. If _nx_ &lt; 0 or _nx_ is *-0*, return an implementation-approximated value representing +&pi;.
+          1. If _ny_ is *-0*, then
+            1. If _nx_ &gt; 0 or _nx_ is *+0*, return *-0*.
+            1. If _nx_ &lt; 0 or _nx_ is *-0*, return an implementation-approximated value representing -&pi;.
+          1. If _ny_ &lt; 0, then
+            1. If _nx_ is *+0* or _nx_ is *-0*, return an implementation-approximated value representing -&pi; / 2.
+          1. If _ny_ &gt; 0 and _ny_ is finite, then
+            1. If _nx_ is *+&infin;*, return *+0*.
+            1. If _nx_ is *-&infin;*, return an implementation-approximated value representing +&pi;.
+          1. If _ny_ &lt; 0 and _ny_ is finite, then
+            1. If _nx_ is *+&infin;*, return *-0*.
+            1. If _nx_ is *-&infin;*, return an implementation-approximated value representing -&pi;.
+          1. If _nx_ is finite, then
+            1. If _ny_ is *+&infin;*, return an implementation-approximated value representing +&pi; / 2.
+            1. If _ny_ is *-&infin;*, return an implementation-approximated value representing -&pi; / 2.
+          1. If _ny_ is *+&infin;*, then
+            1. If _nx_ is *+&infin;*, return an implementation-approximated value representing +&pi; / 4.
+            1. If _nx_ is *-&infin;*, return an implementation-approximated value representing +3&pi; / 4.
+          1. If _ny_ is *-&infin;*, then
+            1. If _nx_ is *+&infin;*, return an implementation-approximated value representing -&pi; / 4.
+            1. If _nx_ is *-&infin;*, return an implementation-approximated value representing -3&pi; / 4.
+          1. Return an implementation-approximated value representing the result of the arc tangent of the quotient _ny_ / _nx_.
```

Comment:
```suggestion
          1. Return an implementation-approximated value representing the result of the inverse tangent of the quotient _ny_ / _nx_.
```

Reply:
I think that this PR needs to be overall consistent with either using "arc" or "inverse"; currently it uses both.